cmake/modules/hunter_windows_env_command.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2015 Sumedh Ghaisas, Ruslan Baratov
+# All rights reserved.
+
+include(hunter_windows_to_snf_filename)
+
+#Set command to change environment variables to 8.3 filenames to windows
+function(hunter_windows_env_command variable_name)
+  
+  hunter_windows_to_snf_filename(${CMAKE_C_COMPILER} "CMAKE_C_COMPILER_SNF")
+  hunter_windows_to_snf_filename(${CMAKE_CXX_COMPILER} "CMAKE_CXX_COMPILER_SNF")
+  hunter_windows_to_snf_filename(${CMAKE_RC_COMPILER} "CMAKE_RC_COMPILER_SNF")
+  
+  set(${variable_name} "set" "cc=${CMAKE_C_COMPILER_SNF}" "&&" "set" "cxx=${CMAKE_CXX_COMPILER_SNF}" "&&" "set" "rc=${CMAKE_RC_COMPILER_SNF}" PARENT_SCOPE)
```

Comment:
Ohh I like this idea. Separate cmake files for configuration, build and install. So we can unset the current variables and set some new. I will shift the boost to this format as this is much cleaner approach. I think hunter should make this approach default for all non cmake builds. Cause any other command can be incorporated in cmake. Also my opinion is to shift the cmake files from scripts to individual folders. Was there any reason behind keeping the qt cmake files in scripts? What do you think about making this a default for non cmake builds?